@shared/api/external.ts

```diff
@@ -68,34 +68,19 @@ export const submitToken = async (args: {
 
 export const submitTransaction = async (
   transactionXdr: string,
-  opts?:
-    | string
-    | {
-        accountToSign?: string;
-        networkPassphrase?: string;
-      },
-  accountToSign?: string,
+  opts?: {
+    address?: string;
```

Comment:
Ah, I see now that all the sign APIs use `address` but this function used `accountToSign` so there was probably a mismatch here